build/Dockerfile

```diff
@@ -67,6 +71,15 @@ RUN cargo chef cook --release --recipe-path recipe.json
 COPY ./offchain/ .
 RUN cargo build --release
 
+#
+# Go build
+#
+FROM golang:${GO_VERSION}-bookworm as go-builder
+ARG GO_BUILD_PATH
+WORKDIR ${GO_BUILD_PATH}
+COPY . .
+RUN go install -ldflags "-s -w" ./cmd/rollups-node
```

Comment:
Shouldn't the binary share the same prefix as others?

`cartesil-rollups-node`